pkg/source/source_test.go

```diff
@@ -211,7 +212,7 @@ var _ = Describe("Source", func() {
 			instance := source.Kind{}
 			f := false
 			Expect(instance.InjectCache(&informertest.FakeInformers{Synced: &f})).To(Succeed())
-			err := instance.WaitForSync(nil)
+			err := instance.WaitForSync(context.Background(), time.Duration(10)*time.Second)
```

Comment:
Do we need to pass in a different duration? We default to 10, what happens if this is > 10?